.circleci/config.yml

```diff
@@ -30,6 +30,22 @@ references:
     attach_workspace:
       at: *hermes_workspace_root
 
+  # -------------------------
+  #        Cache Key Anchors
+  # -------------------------
+  # Anchors for the cache keys
+
+  cache_keys:
+    gems_cache_key: &gems_cache_key v1-gems-{{ checksum "Gemfile.lock" }}
```

Comment:
I see that the `gem cache` key was [previously](https://github.com/facebook/react-native/pull/33821/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L99) only `1-`, now I see that it is `v1-`. Was it inteded or the force of habit?

Reply:
Sorry, I forgot to add a comment here, this is the only one that was missing the v, so I've added it here as well :)

Reply:
Correct, but from that moment on it will always find the key